migration/src/main/resources/META-INF/rewrite/axon-javax.yml

```diff
@@ -0,0 +1,55 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.axonframework.axon.UpgradeAxonFramework_4_7_Javax
+displayName: Upgrade to Axonframework 4.7 Javax
+description: This is what you may want if you don’t want Spring 6 and Spring Boot 3 support and expect things to stay the same.
+recipeList:
+  # Move all classes from org.axonframework.common.jpa to org.axonframework.common.legacyjpa
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: org.axonframework.common.jpa
+      newPackageName: org.axonframework.common.legacyjpa
+  # Move selected classes
```

Comment:
I would expect the `org.axonframework.common.jpa.PagingJpaQueryIterable` to `org.axonframework.common.legacyjpa.PagingJpaQueryIterable` too here.

Reply:
Ah! I see you've covered that with the `ChangePackage` step above (right?)! Nice!

Reply:
Yes, when all classes in a package have moved, you can use the `ChangePackage` to update the references. Keeps this yml a little more sane.